pillar/rabbitmq/init.sls

```diff
@@ -7,9 +7,6 @@ rabbitmq:
   overrides:
     version: '3.7.4-1'
     erlang_version: '1:20.1'
-  environment:
-    RABBITMQ_CTL_DIST_PORT_MIN: 35672
-    RABBITMQ_CTL_DIST_PORT_MAX: 35672
```

Comment:
So should this be `35682`?

Reply:
This is removing those lines so that it will just use the defaults.